### DISCUSSION TITLE
Some internal images are unable to open: Invalid argument @ error/blob.c/OpenBlob/3537.

### DISCUSSION THREAD

**Discussion by ooker777** [ImageMagick/ImageMagick#5972] [General]
I try several internal images like `script:`, `red-ball:`, `wand:`, `bricks:` and they all return this error:
```PowerShell
magick.exe: unable to open image 'bricks:': Invalid argument @ error/blob.c/OpenBlob/3537.
magick.exe: no decode delegate for this image format `' @ error/constitute.c/ReadImage/572.
```

Some other internal images like `logo:`, `wizard:`, `rose:` work fine.

The command I use is a simple `magick logo: image.jpg`

v7.0.11-14 on Windows 11.

**Comment by dlemstra** [comment]
Those formats are read differently. You should use `magick pattern:bricks bricks.jpg` instead. And next time please open a discussion instead.

**Comment by ooker777** [comment]
Is there a documentation for these? I try `magick pattern:red-ball image.jpg` and it says 
```
magick.exe: unrecognized image format `red-ball' @ error/pattern.c/ReadPATTERNImage/974.
```

Also, I thought that this can be converted to discussion easily?

**Reply by dlemstra** [reply]
You can find a complete list here: https://imagemagick.org/script/formats.php#builtin-images.

And yes it can be converted easily just wanted to let you know that you should use a discussion next time.

**Comment by fmw42** [comment]
For patterns, see https://imagemagick.org/script/formats.php#builtin-patterns
